package.json

```diff
@@ -1,29 +1,33 @@
 {
-  "name": "lwc2react",
-  "version": "0.1.4",
+  "name": "root",
+  "private": true,
   "description": "",
   "main": "lib/index.js",
   "homepage": "https://github.com/blittle/lwc2react",
   "scripts": {
-    "test": "jest --verbose --coverage",
-    "prepublish": "npm run build",
-    "build": "babel src -d lib"
+    "build": "lerna run build",
+    "format": "prettier --write '**/*.{js,json}' '!{node_modules,packages}/**' '!package*.json' && lerna run format",
+    "lint": "lerna run lint",
+    "test": "lerna run test",
+    "version": "rm -f ./packages/**/CHANGELOG.md && npm run format"
```

Comment:
Should command also proxy `lerna version`?

Reply:
So the intention here is different. Lerna under the hood triggers npm's version hook after it's own version command has been executed. What this version hook now does is to get some stuff right that would otherwise be inconvenient:
* Lerna creates separate changelog files per package which is not what we need as they are empty anyways, so I simply remove them
* Lerna tends to update the package.json files with a different format than prettier does. So in order to not have a war of tools with formatting ping pong, I just run the format script to have the `lerna version` command in the end come up with only the actually required changes.